websites/L/LiveLinkBio/presence.ts

```diff
@@ -0,0 +1,90 @@
+const presence = new Presence({
+		clientId: "966711989533544580"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp
+		},
+		[privacy, buttons] = await Promise.all([
+			presence.getSetting<boolean>("privacy"),
+			presence.getSetting<boolean>("buttons")
+		]),
+		search = document.querySelector<HTMLInputElement>(
+			"#portals-root > div:nth-child(3) > div > div > div > div > div.css-1dbjc4n.r-1ro0kt6.r-16y2uox.r-1wbh5a2 > input"
+		);
+	let title: string;
+	if (privacy) presenceData.details = "Browsing...";
+	else if (document.location.hostname.includes("docs.livelinkbio.com")) {
+		if (search?.value) {
+			presenceData.details = "Searching for:";
+			presenceData.state = search.value ?? "Unknown";
```

Comment:
you check if search.value is truthy above so no need to have a nullish check here
```suggestion
			presenceData.state = search.value;
```